package-lock.json

```diff
@@ -33932,6 +33932,21 @@
     "tooling/typescript": {
       "name": "@isomer/tsconfig",
       "version": "0.0.0"
+    },
+    "tooling/template/node_modules/@next/swc-win32-ia32-msvc": {
```

Comment:
whats this change

Reply:
i'm not very sure - this was something auto installed/updated when i did `npm i`.

based off what i googled (nextjs issue [here](https://github.com/vercel/next.js/discussions/61407)), this appears to be a nextjs binary for win32 (specifically for swc, a js runtime). 

i can't be sure about this, but i suspect this (swc) is used under the hood by next and this got updated as part of deps

Reply:
yeah can confirm - this is what's autoinstalled during npm i, it's also stated in the console

Reply:
sounds good thanks for confirming